cmd/vita/main.go

```diff
@@ -25,3 +41,61 @@ func main() {
 		universe.Tick()
 	}
 }
+
+func runParallelUniverses(multi []*game.ParallelUniverse) {
+	for i := 0; i < *generations; i++ {
+		for i, u := range multi {
+			fmt.Println("Universe", i)
+			fmt.Println(u)
+			fmt.Println()
+		}
+
+		var wg sync.WaitGroup
+		for _, u := range multi[:len(multi)-1] {
+			callMultiTick(&wg, u)
+		}
+		wg.Wait()
+
+		multi[len(multi)-1].MultiTick()
```

Comment:
question: why is this needed?

Reply:
It isn't actually. I just consolidated that into the main loop.